src/WpfMath/Atoms/BigOperatorAtom.cs

```diff
@@ -106,28 +106,28 @@ protected override Box CreateBoxCore(TexEnvironment environment)
             if (this.UpperLimitAtom != null)
             {
                 resultBox.Add(new StrutBox(0, opSpacing5, 0, 0));
-                upperLimitBox.Shift = delta / 2;
+                upperLimitBox!.Shift = delta / 2;
                 resultBox.Add(upperLimitBox);
                 kern = Math.Max(texFont.GetBigOpSpacing1(style), texFont.GetBigOpSpacing3(style) -
                     upperLimitBox.Depth);
                 resultBox.Add(new StrutBox(0, kern, 0, 0));
             }
 
             // Add box for base atom.
-            resultBox.Add(baseBox);
+            resultBox.Add(baseBox!);
```

Comment:
Interestingly, the compiler complains about it being potentially `null`. Seems like a bug. And it's apparently triggered by the following `if`:

```
            if (baseBox != null)
                baseBox = ChangeWidth(baseBox, maxWidth);
```

If I make the `ChangeWidth` call unconditional (since `baseBox` can't be `null`), this and the following diagnostic goes away.

Reply:
Yeah, I've already inspected this curiosity. It is okay to drop the condition here.